CHANGELOG.md

```diff
@@ -2,11 +2,32 @@
 
 ## v2.0.0
 
+* Breaking change from 1.x: Removed deprecated methods
+  `is_in_european_union`, `is_anonymous`, `is_anonymous_vpn`,
+  `is_hosting_provider`, `is_public_proxy`, and `is_tor_exit_node`. The
+  non-deprecated equivalents are `in_european_union?`, `anonymous?`,
+  `anonymous_vpn?`, `hosting_provider?`, `public_proxy?`, and
+  `tor_exit_node?`.
+* Breaking change from 1.x: Removed deprecated methods for deprecated
+  attributes: `is_anonymous_proxy` and `is_satellite_provider`.
```

Comment:
These are what I was referring to when I said, "The one exception that we have made for the other libraries are the deprecated GeoIP2 outputs as the web service still sends them."  These outputs could have some value so I am a bit hesitant about removing them before the web service does.

Note, this does not apply to `email_tenure` and `ip_tenure`. I don't think we send those anymore.

Reply:
I see. Thanks! I wonder if we should put them into GeoIP2-ruby rather than here then actually. They aren't accessible using that gem currently.

Reply:
I dropped the commit that deleted them and added a commit with some explanation about why we're keeping them!